src/IndexedDB/Browser.IndexedDB.fs

```diff
@@ -0,0 +1,143 @@
+﻿namespace rec Browser.Types
+
+open System
+open Fable.Core
+
+
+[<Erase>]
+type IDBRequestSource =
+    | Index of IDBIndex
+    | ObjectStore of IDBObjectStore
+    | Cursor of IDBCursor
+
+[<Erase>]
+type IDBRequestReadyState =
+    | Pending
+    | ``Done``
```

Comment:
Normally we use `StringEnum` for these cases, but when the cases have no fields `Erase` and `StringEnum` have the same effect, so it should be fine 👍

Reply:
Good to know!